src/services/item/fluent-schema.ts

```diff
@@ -159,8 +160,16 @@ export const getParents = {
 };
 
 export const getOwn = {
+  querystring: S.object()
+    .additionalProperties(false)
+    .prop('page', S.number().default(1))
+    .prop('name', S.string())
+    .prop('limit', S.number().default(ITEMS_LIST_LIMIT)),
```

Comment:
Is it desirable to let the consumer set an arbitrary limit? Should it be capped? @morganridel @pyphilia

Reply:
We still need the full list for some cases, but I agree it should be capped sometime. @LinaYahya could you add a comment about that?

Reply:
Not sure how we would cap the value <sometime>, the point of we can't set a limit for `limit`, in case of getting the whole list we will send big values! for this we can't do have a maximum bound for it.
@pyphilia  @dialexo